cli/ballerina-cli/src/main/resources/cli-help/ballerina-graphql.help

```diff
@@ -21,11 +26,12 @@ DESCRIPTION
 
 
 OPTIONS
-        -i, --input <graphql-configuration-file-path | ballerina-graphql-service-file-path>
+        -i, --input <graphql-configuration-file-path | ballerina-graphql-service-file-path | graphql-schema-file-path>
```

Comment:
Is there any character limit for a line here @DimuthuMadushan ?

Reply:
@lnash94 @udda1996 any idea?